### PR TITLE
Consistent research window view behaviour

### PR DESCRIPTION
**Pull request by dbenage-cx**
Addresses the bulk of issues raised in #1628.

color/border changes and addition of "Move to Top of Queue" for already queued techs are not addressed in this PR.

### CODE REVIEW

File: UI/TechTreeWnd.cpp [hunk 1]
```diff
@@ -123,17 +123,17 @@ namespace {
...
+std::shared_ptr<GG::BrowseInfoWnd> TechRowBrowseWnd(const std::string& tech_name, int empire_id) {
     const Empire* empire = GetEmpire(empire_id);
     const Tech* tech = GetTech(tech_name);
     if (!tech)
         return nullptr;
 
     std::string main_text;
 
-    main_text += UserString(tech->Category()) + " ";
+    main_text += UserString(tech->Category()) + " \u2013 ";
```
Comment (geoffthemedio) [inline UI/TechTreeWnd.cpp]: document `\u2013`

Comment (dbenage-cx) [reply UI/TechTreeWnd.cpp]: Fixed, please note if `u2013 = 'En dash'` is not sufficient.

**Comment by LGM-Doyle** [comment]
I've tested this PR and all the features work as intended.

Right-click menu add to top, add to bottom and pedia lookup work.
Ctrl-double click works.
Browse info windows work, and have the dash.

Code looks good as well.

**Comment by dbenage-cx** [comment]
@LGM-Doyle TY for testing

**Pull request merged**